### PR TITLE
Updated visual.j

### PR DESCRIPTION
**Pull request by Jaccouille**

### CODE REVIEW

**Comment by Luashine** [comment]
iirc the parser expects all \@tags to be at the bottom at the moment. @lep confirm please? Basically you cannot have:

    @patch 1.32
    ... description below the tag

Instead change to:
 
    ... description below the tag
    @patch 1.32

File: visuals.j [hunk 1]
```diff
@@ -245,6 +250,15 @@ native BlzGetItemSkin                                 takes item whichItem retur
 // native BlzGetDestructableSkin                         takes destructable whichDestructable returns integer
 
 /**
+Replaces a unit's model with the unit's model referenced by the skinId.
+BlzSetUnitSkin(whichUnit, 'hfoo') will replace whichUnit model with the footman one.
```
Comment (lep) [inline visuals.j:254]: Put the code inside ``` `backticks` ```.

File: visuals.j [hunk 2]
```diff
@@ -262,6 +276,17 @@ native BlzSetItemSkin                                 takes item whichItem, inte
 native BlzCreateItemWithSkin                       takes integer itemid, real x, real y, integer skinId returns item
 
 /**
+Creates a unit with the model from the unit referenced by the skinId.
+BlzCreateUnitWithSkin(players[0], 'hpea', 0, 0, 270, 'hfoo') will create a peasant with a footman model.
```
Comment (lep) [inline visuals.j:280]: Put the code inside ``` `backticks` ```.

**Review by lep — COMMENTED** [review]
Thanks for the contribution. I added two small comments.

**Pull request merged**